sdk/formrecognizer/Azure.AI.FormRecognizer/tests/SessionRecords/FormRecognizerClientLiveTests/StartRecognizeCustomFormsWithoutLabelsCanParseMultipageForm(True).json

```diff
@@ -8,8 +8,8 @@
         "Content-Length": "43",
```

Comment:
why was this file re-recorded? like what changed that made it re-record? same for the async version of it

Reply:
We're re-recording the tests in which we were using the recently sanitized environment variable. It's not valid anymore, so not a big deal. There's still value in re-recording it to avoid false positives in the future when we have a reliable system to detect them.

Reply:
True for the other files, but this one already has the sanitized SAS in [master](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/SessionRecords/FormRecognizerClientLiveTests/StartRecognizeCustomFormsWithoutLabelsCanParseMultipageForm(True).json#L4930), so not sure why we need to re-record it again

Reply:
You are right. Completely forgot we needed to update this test because of the service behavior change. Let me update the PR.

Reply:
No worries. I wanted to make sure there wasn't something new I wasn't aware of that we changed too and that caused the re-recording